test/helpers/currency-format.test.js

```diff
@@ -14,6 +14,20 @@ test('currencyFormat() properly processes a monetary currency', () => {
   expect(result).toBe('€ 25.43')
 })
 
+test('currencyFormat() properly processes a monetary currency with support for astlijntje', () => {
+  const options = {
+    isMoney: true,
+    locale: 'de-DE',
+    abbreviation: 'EUR',
+    decimalTerminator: '-'
+  }
+
+  const result = helperFunction(25, options)
+
+  expect(typeof result).toBe('string')
+  expect(result).toBe('€ 25.-')
```

Comment:
It only makes sense in Dutch though, so that would be `€ 25,-`, but for testing this is fine of course.

Reply:
I know. But Node.js does not always come installed with internationalisation prefs for the comma. So for example, if Travis does not have those, this test will fail. Since we're specifically testing the behaviour of the `decimalTerminator`, there's no point I think in hustling to make sure the right localised Node.js is installed in the Travis container.

Reply:
I totally agree.